libs/langchain/langchain/output_parsers/openai_tools.py

```diff
@@ -22,6 +22,8 @@ class JsonOutputToolsParser(BaseGenerationOutputParser[Any]):
     """
     return_id: bool = False
     """Whether to return the tool call id."""
+    return_single: bool = False
```

Comment:
Do we think that `return_single` is generic enough to warrant introducing it into the parser? Is this the right place? How does this interact with streaming? Would unpacking at the caller code work?

----

Also if we do keep it here is there a better name? e.g., `first_tool_only`?

The doc-string should probably explain why this is necessary?